proposals/compatibility-mode.md

```diff
@@ -180,14 +178,14 @@ and/or `@builtin(instance_index)` each count as an
 attribute.
 
 Note: Some of the limits are derived from a survey of OpenGL ES 3.1 devices
-and are higher than the limit specified in the OpenGL ES 3.1 spec.
-
-For example, in OpenGL ES 3.1, GL_MAX_FRAGMENT_IMAGE_UNIFORMS and GL_MAX_VERTEX_IMAGE_UNIFORMS can be
-zero but `maxStorageTexturesPerShaderStage` is 4 above as all 3.1 devices support at
-least 4 of each.
```

Comment:
Removed this note because it was obsolete (we have separate limits for these now).